tests/general/QuotesBearTest.py

```diff
@@ -97,3 +99,21 @@ def test_invalid_quotes(self):
                              '-"a string with double quotes!"\n'
                              "+'a string with double quotes!'\n"
                              " 'A single quoted string with \" in it'\n")
+
+        self.section['preferred_quotation'] = '"'
+        self.force_preferred_quotation = True
+        with execute_bear(self.uut, self.filename, self.double_quote_file,
+                          dependency_results=self.dep_results,
+                          force_preferred_quotation=self.
+                          force_preferred_quotation) as results:
```

Comment:
please also test if `force_preferred_quotation=True`, I mean that's what you intend to test, right?

Reply:
(also not necessary imo to put this into a test-wide variable, just do inline `False`^^)

Reply:
It is good to have one right?